string header
cd_harmony/waypoint_msg[] waypoints
cd_harmony/waypoint_group[] groups
